src/hexo/themes/documentation/source/images/results-a11y.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 72 72" style="enable-background:new 0 0 72 72;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#ACADDA;}
+	.st1{fill:#2D28A1;}
+</style>
+<title>icons</title>
+<g>
+	<path class="st0" d="M10.1,26.3l-3.1-3.1c-3.7,4-5.8,9.2-6.1,14.6h4.3C5.6,33.6,7.3,29.5,10.1,26.3z M23.2,20v-4.3
+		c-5.4,0.2-10.6,2.4-14.6,6l3.1,3.1C14.8,21.9,18.9,20.2,23.2,20z M25.3,57.8v4.3c5.4-0.2,10.6-2.4,14.6-6l-3.1-3.1
+		C33.6,55.9,29.6,57.6,25.3,57.8z M11.6,53.1l-3.1,3.1c4,3.7,9.2,5.8,14.6,6v-4.3C18.9,57.6,14.8,55.9,11.6,53.1z M5.3,40H1
+		c0.2,5.4,2.4,10.6,6,14.6l3.1-3.1C7.3,48.3,5.6,44.3,5.3,40z M43.2,40c-0.2,4.3-1.9,8.3-4.8,11.5l3.1,3.1c3.7-4,5.8-9.2,6-14.6
+		H43.2z"/>
+	<path class="st1" d="M29.5,29.4V9.8h-4.3v19.6h-2.6l4.7,8.2l4.7-8.2H29.5z M58.6,35.1l-8.2-4.7v2.6H32.6v4.3h17.8v2.6L58.6,35.1z"
+		/>
+</g>
+</svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 72 72"><path d="M10.1 26.3L7 23.2c-3.7 4-5.8 9.2-6.1 14.6h4.3c.4-4.2 2.1-8.3 4.9-11.5zM23.2 20v-4.3c-5.4.2-10.6 2.4-14.6 6l3.1 3.1c3.1-2.9 7.2-4.6 11.5-4.8zm2.1 37.8v4.3c5.4-.2 10.6-2.4 14.6-6L36.8 53c-3.2 2.9-7.2 4.6-11.5 4.8zm-13.7-4.7l-3.1 3.1c4 3.7 9.2 5.8 14.6 6v-4.3c-4.2-.3-8.3-2-11.5-4.8zM5.3 40H1c.2 5.4 2.4 10.6 6 14.6l3.1-3.1C7.3 48.3 5.6 44.3 5.3 40zm37.9 0c-.2 4.3-1.9 8.3-4.8 11.5l3.1 3.1c3.7-4 5.8-9.2 6-14.6h-4.3z" fill="#acadda"/><path d="M29.5 29.4V9.8h-4.3v19.6h-2.6l4.7 8.2 4.7-8.2h-2.5zm29.1 5.7l-8.2-4.7V33H32.6v4.3h17.8v2.6l8.2-4.8z" fill="#2d28a1"/></svg>
```

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?